### DISCUSSION TITLE
Welcome to CT-image-segmentation-of-pulmonary-embolism Discussions!

### DISCUSSION THREAD

**Discussion by FeijiangHan** [FeijiangHan/CTISPED#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cuuuuu123** [comment]
Hello,when I use your code--UNet 2D to predict my test dataset, it always was error. So, I want to know how the form of the test dataset is and the images size should resize? Thank you.

**Reply by FeijiangHan** [reply]
Hi, the original dataset I used is the open-source pulmonary embolism detection dataset https://tianchi.aliyun.com/dataset/90713. I preprocessed it into 512x512 PNG images (for both data and labels).

![image](https://github.com/FeijiangHan/CTISPED/assets/88610657/f6358262-20ea-4028-a653-c67797356427)

**Reply by Cuuuuu123** [reply]
Oh, it's kind of you to reply my question. I do really want to know what the mean of this code in the generate_predict.py or generate_metric.py:
**org_mask = cv2.imread(image.replace('image', 'label'), 0)**
Dose it mean I need two files like my picture include images and masks? Or I have to take the images and masks in the same file instead of separating them. And I make sure that they have the same name and have resized the same size.
Thank you for your kindness reply me again.
![屏幕截图 2024-04-07 101923](https://github.com/FeijiangHan/CTISPED/assets/166238404/43318f27-73e3-4cb3-ad60-063e195177a9)
